composer/models/bert/model.py

```diff
@@ -45,12 +45,20 @@ class BERTModel(ComposerTransformer):
         model = BERTModel(module=hf_model, config=config, tokenizer=tokenizer)
     """
 
-    def __init__(self, module: transformers.BertModel, config: transformers.BertConfig,
-                 tokenizer: transformers.BertTokenizer) -> None:
+    def __init__(self,
+                 module: transformers.BertModel,
+                 config: transformers.BertConfig,
+                 tokenizer: Optional[transformers.BertTokenizer] = None) -> None:
+
+        if tokenizer is None:
+            model_inputs = {"input_ids", "attention_mask", "token_type_ids"}
+        else:
+            model_inputs = set(tokenizer.model_input_names)
```

Comment:
Not recommending changing this now, but this seems to suggest that the BERTModel has a set of required input names, would it make sense to validate those like `assert "input_ids" in tokenizer.model_input_names` ? If I passed a tokenizer with the wrong input names, would the rest of the model crash?